### PR TITLE
Music: share updates

### PR DESCRIPTION
**Pull request by breville**
More updates to the share dialog, both for the congrats and share cases, primarily to get updated strings ready for localization.  Follows up work in https://github.com/code-dot-org/code-dot-org/pull/61644.

#### Music Lab congrats

Screenshot 2024-10-21 at 9 22 26 PM

#### Share

Screenshot 2024-10-21 at 7 14 51 PM

### CODE REVIEW

File: apps/src/lab2/views/dialogs/ShareDialog.module.scss [hunk 1]
```diff
@@ -61,19 +51,47 @@
...
       flex-direction: column;
       gap: 20px;
+      height: 100%;
+
+      &Heading {
+        text-align: center;
+      }
+
+      &Afe {
+        background-color: #2E4486;
```
Comment (sanchitmalhotra126) [inline apps/src/lab2/views/dialogs/ShareDialog.module.scss:68]: Out of curiosity is this a color provided by AFE? Or can we use one of our constants?

File: apps/src/lab2/views/dialogs/ShareDialog.tsx [hunk 2]
```diff
@@ -28,17 +29,49 @@ const CopyToClipboardButton: React.FunctionComponent<{
...
+  return (
+    <div className={classNames(moduleStyles.block, moduleStyles.blockAfe)}>
+      <Typography
+        semanticTag="h2"
+        visualAppearance="heading-md"
+        className={classNames(moduleStyles.heading, moduleStyles.blockHeading)}
+      >
+        {i18n.careerTourTitle()}
+      </Typography>
+      <img alt="" src="/shared/images/afe/afe-career-tours-0.jpg" />
```
Comment (sanchitmalhotra126) [inline apps/src/lab2/views/dialogs/ShareDialog.tsx:60]: Alt text for this image?

**Review by sanchitmalhotra126 — APPROVED** [review]
Looks good!

I'd be curious to see how the share dialog looks if the width of the share card was constrained to the same width as it is in the congrats dialog but remained centered?

**Comment by dju90** [comment]
Just making a note here so we don't forget: can we add the "Did you like this activity?" string to this PR too to get translated? Thanks!

**Comment by breville** [comment]
> I'd be curious to see how the share dialog looks if the width of the share card was constrained to the same width as it is in the congrats dialog but remained centered?

Good question.  I think we'll probably end up trying this in the next round.

File: apps/src/lab2/views/dialogs/ShareDialog.module.scss (hunk 1 above)
Comment (breville) [reply apps/src/lab2/views/dialogs/ShareDialog.module.scss:68]: It came from an internal design comp, and I didn't see a match.  Let's keep an eye on this as we make revisions.

File: apps/src/lab2/views/dialogs/ShareDialog.tsx (hunk 2 above)
Comment (breville) [reply apps/src/lab2/views/dialogs/ShareDialog.tsx:60]: From reading a couple articles, it seems we don't need alt text if the nearby text conveys all the necessary information, which I think is the case here.

**Comment by breville** [comment]
> I'd be curious to see how the share dialog looks if the width of the share card was constrained to the same width as it is in the congrats dialog but remained centered?

Good question.  I've updated with that approach now.

**Comment by dju90** [comment]
Here's a high-resolution version of the image to use
AFE Tours_HoC

**Comment by dju90** [comment]
Updated description copy from AFE:
> Explore more careers in tech and music with Amazon.

**Pull request merged**